src/api-reference/invoice/v3.purchase-order-receipt.markdown

```diff
@@ -0,0 +1,180 @@
+* [Schema](#schema-receipt)
+* [Response schema](#schema-response)
+* [Receipt schema](#schema-receipt)
+* [Error codes](#error-codes)
+
+## Version
+
+3.0  
+
+## <a name="post"></a>Create a new purchase order receipt 
+
+```
+POST /api/v3.0/invoice/purchaseordersreceipts
+```
+
+Creates purchase order receipts and associate if to purchase order line item and returns status of updation.
+
+### <a name="8"></a>Parameters  
+
+Name|Type|Format|Description
+---|---|---|---
+`purchaseOrderReceipt`|-|`body`|The details of the purchase order receipt.
+
+### Input  
+
+[Schema](#schema-receipt)
+
+```json
+{
+  "Deleted": false,
+  "DeliverySlipNumber": "DSN1",
+  "GoodsReceiptNumber": "RCPT1",
+  "ID": "1000001",
+  "LineItemExternalID": "LN000001",
+  "PurchaseOrderNumber": "PO12345",
+  "ReceivedDate": "2018-09-15 00:00:00.0",
+  "ReceivedQuantity": "14.5",
+  "UnitOfMeasureCode": "DA",
+  "URI": "string",
+  "Version": "2.0"
+}
+```
+
+### Response  
+
+[Response schema](#schema-response)
+
+## <a name="put-receipts"></a>Update purchase order line item with receipt information  
+
+```
+PUT /api/v3.0/invoice/purchaseorderreceipts
+```
+### Parameters  
+
+Name|Type|Format|Description
+---|---|---|---
+`purchaseOrderReceipt`|-|`body`|Purchase order receipt information.
+
+### Input  
+
+[Receipt schema](#schema-receipt)
+
+### Response  
+
+[Response schema](#schema-response)
+
+
+## <a name="get"></a>Get existing purchase order receipt 
+
+```
+GET /api/v3.0/invoice/purchaseorderreceipts?purchaseOrderNumber=PO1&lineItemExternalID=L1&goodsReceiptNumber=GRN1
+```
+
+### Parameters  
+
+Name|Type|Format|Description
+---|---|---|---
+`purchaseOrderNumber`|`string`|{purchaseOrderNumber}|The purchase order number.
+`lineItemExternalID`|`string`|{lineItemExternalID}|A customer-supplied value that uniquely identifies the line item within the purchase order.
+`goodsReceiptNumber`|`string`|{goodsReceiptNumber}|Goods receipt number for a purchase order line item receipt.
+
+
+### Input  
+
+None.
+
+### Response
+
+[Schema](#get-schema-receipt)
+
+## <a name="delete"></a>Delete existing purchase order receipt
+
+```
+DELETE /api/v3.0/invoice/purchaseorderreceipts?purchaseOrderNumber=PO1&lineItemExternalID=L1&goodsReceiptNumber=GRN1
+```
+
+### Parameters  
+
+Name|Type|Format|Description
+---|---|---|---
+`purchaseOrderNumber`|`string`|{purchaseOrderNumber}|The purchase order number.
+`lineItemExternalID`|`string`|{lineItemExternalID}|A customer-supplied value that uniquely identifies the line item within the purchase order.
+`goodsReceiptNumber`|`string`|{goodsReceiptNumber}|Goods receipt number for a purchase order line item receipt.
+
+
+### Input  
+
+None.
+
+### Response
+
+[Response schema](#schema-response)
+
+## <a name="schema-response"></a>Response schema  
+
+Name|Type|Format|Description
+---|---|---|---
+`ErrorCode`|`string`|-|A code that indicates why the purchase order was not processed successfully.
```

Comment:
Should this be purchase order receipt instead of purchase order?

Reply:
Fixed in bbd8fe90